classrooms/models.py

```
@@ -1,3 +1,83 @@
+    total_rating = models.DecimalField(max_digits=5, decimal_places=2, default=0.0)
+    number_of_ratings = models.PositiveIntegerField(default=0)
+
+    def __str__(self):
+        return self.title
+
+    def average_rating(self):
+        if self.number_of_ratings > 0:
+            return self.total_rating / self.number_of_ratings
+        return 0.0
+
+    def save(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for public function `save`

Add return type annotation: `None`

(ANN201)